superset/assets/javascripts/explorev2/actions/exploreActions.js

```diff
@@ -85,3 +87,39 @@ export function changeFilterValue(filter, value) {
 export function setFieldValue(key, value) {
   return { type: SET_FIELD_VALUE, key, value };
 }
+
+export function updateViz(viz) {
+  return { type: UPDATE_VIZ, viz };
+}
+
+export const CHART_UPDATE_STARTED = 'CHART_UPDATE_STARTED';
```

Comment:
it's a nice pattern to put the action name at the beginning: `UPDATE_CHART_STARTED`